scripts/check-release-please.mjs

```diff
@@ -29,35 +29,48 @@ const getProcessOutput = (cmd, args) => {
 	return result.stdout.toString('utf8');
 }
 
-const lernaList = JSON.parse(
-	getProcessOutput('npx', ['lerna', 'list', '--json'])
-);
+const lernaList = JSON
+	.parse(getProcessOutput('npx', ['lerna', 'list', '--json', '-a']))
+	.map((pkgInfo) => {
+		pkgInfo.relativeLocation = path.relative(PROJECT_ROOT, pkgInfo.location);
+		return pkgInfo;
+	});
 const manifest = readJson('.release-please-manifest.json');
 const config = readJson('release-please-config.json');
 
 const lernaPackages = new Set(
-	lernaList.map((pkg) => {
-		return path.relative(PROJECT_ROOT, pkg.location);
-	})
+  lernaList.map((pkgInfo) => pkgInfo.relativeLocation)
 );
 const manifestPackages = new Set(Object.keys(manifest));
 const configPackages = new Set(Object.keys(config.packages));
 
-console.log('lerna packages', lernaPackages);
-console.log('manifest packages', manifestPackages);
-console.log('config packages', configPackages);
-
-lernaPackages.forEach((relativeLocation) => {
-	logErrorIf(
-		!manifestPackages.has(relativeLocation),
-		`Could not find ${relativeLocation} in .release-please-manifest.json. If you are adding a new package. Add following
-  "${relativeLocation}": "0.0.1",`);
+// console.log('lerna list', lernaList);
+// console.log('manifest packages', manifestPackages);
+// console.log('config packages', configPackages);
```

Comment:
Did you mean to check in commented code?

Reply:
Kinda/sorta?  These *had* been outputs of the script before my changes here. I'll remove them.